automation_testing/tests/form/form_with_layout.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from "@playwright/test";
+import { FormWithLayout } from "../../pages/Form/FormWithLayoutPage";
+
+test.describe("Form - Form With Layout", () => {
+	let formWithLayoutPage: FormWithLayout;
+
+	test.beforeEach(async ({ page }) => {
+		formWithLayoutPage = new FormWithLayout(page);
+		await formWithLayoutPage.visit("form_with_layout");
+	});
+
+	test("Validate that each Section has a Title", async () => {
+		const expectedNumberOfTitles = await formWithLayoutPage.getNumberOfSectionsFromTopComponent();
+		const numberOfSections = await formWithLayoutPage.getNumberOfSections();
+		expect(numberOfSections).toBe(expectedNumberOfTitles);
+		expect(await formWithLayoutPage.getSectionsTitles()).not.toBe("");
+	});
+
+	test("Validate that each Section has a Description", async () => {
+		expect(await formWithLayoutPage.getSectionsDescriptions()).not.toBe("");
+	});
+
+	test("Validate that each row in each section does not have more than 3 columns", async () => {
+		const numberOfColumnsPerRow = await formWithLayoutPage.getNumberOfColumsInEachRowFromSections()
```

Comment:
This line seems to be missing a ";" that might cause issues on our github actions.